src/EventEdition/GuestGrid.vala

```diff
@@ -106,8 +108,17 @@ public class Maya.View.EventEdition.GuestGrid : Gtk.Grid {
                         individual = map_iterator.get_value ();
                         if (individual != null) {
                             try {
-                                individual.avatar.load (32, null);
-                                avatar = new Granite.Widgets.Avatar.from_file (individual.avatar.to_string (), 32);
+                                individual.avatar.load (ICON_SIZE, null);
+                                avatar = new Hdy.Avatar (ICON_SIZE, individual.avatar.to_string (), true);
+                                avatar.set_image_load_func (() => {
```

Comment:
You're not taking the scale facto into account here, I think that the intended behavior should be something like:
```suggestion
avatar.set_image_load_func ((size) => {
    try {
        return new Gdk.Pixbuf.from_stream_at_scale (individual.avatar.load (size, null), size, size, true);
    } catch (Error e) {
        debug (e.message);
        return null;
    }
});
```

Reply:
Whops, you're right. Thanks for noticing. 😅️ I'll push these changes now.